utils.py

```diff
@@ -61,11 +61,11 @@ def _read_builds():
 
 
 def get_requested_llvm_version():
-    ver = int(os.environ["LLVM_VERSION"])
+    ver = str(os.environ["LLVM_VERSION"])
     ci_folder = pathlib.Path(__file__).resolve().parent
     with open(ci_folder.joinpath("LLVM_TOT_VERSION")) as f:
-        llvm_tot_version = int(f.read())
-    return "clang-" + ("nightly" if ver == llvm_tot_version else str(ver))
+        llvm_tot_version = str(f.read()).strip()
```

Comment:
perhaps still worth validating that the contents of this file can be converted to `int`?

Reply:
```llvm_tot_version = str(int(f.read())).strip()```?

Comparing `int` to `str` won't work.

Reply:
yes; that way if we put something that can't be parsed as an int, it will fail loudly (louder? maybe doesn't matter then)

Reply:
Done.